src/webgl/p5.RendererGL.js

```diff
@@ -2545,4 +2557,24 @@ p5.prototype._assert3d = function (name) {
 
 p5.RendererGL.prototype.tessyVertexSize = 12;
 
+p5.RendererGL.prototype.background = function (...args) {
```

Comment:
It looks like we have a duplicate implementation here that overrides the method inside the class above, are we able to merge this with the implementation above? It looks like this is the code currently being executed, so we could replace the implementation above with the body of this function and then remove it.